templates/rhsm-katello-reconfigure.erb

```diff
@@ -95,6 +95,11 @@ fi
   service goferd status >/dev/null && \
   service goferd restart >/dev/null 2&>1
 
+
+if [ -d /etc/rhsm/facts/ ]; then
+  echo "{\"network.hostname-override\":\"`hostname -f`\"}" > /etc/rhsm/facts/katello.facts
```

Comment:
It was meant more as a temporary workaround fact that only exists to help with issue.  The fact it also never would actually be updated further reinforces to me that the name should indicate that its not a 'real' fact that should be depended on by users. I wouldn't want to see:

hostname: foo
fqdn:  foo.example.com
hostname-full:  someother.name.that.is.no.longer.valid

versus:
hostname-override: someother.name.that.is.no.longer.valid

not saying hostname-override is a fantastic name, but i figured it should not appear 'normal'.